### ISSUE TITLE
Global header search button is not accessible to keyboard users

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#26530]
##  Description
As Described in [#26416](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26416)

❗ Launchblocker Global header search button is not accessible to keyboard users.
Screen reader users will be instructed to use ctrl-option-space to activate it, but the button will not respond unless a mouse click is used.

## Acceptance Criteria
- [x] Comment in [#26416](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26416) to make Josh aware to validate
- [x] Tier 3 (No Socks validation necessary)

**Comment by srsuddath** [comment]
Hey @joshkimux can you provide some clarity for me on this issue?
1. Which screen reader are you using?
2. what is the command you are referencing? The functionality currently works with the user pressing "enter" to initiate a search, and I am wondering if we can update that "call out" instead, or if it is a particular screen reader specific issue

**Comment by joshkimux** [comment]
@srsuddath the main issue is that the search button doesn't work when the user presses `space` which is a common input for keyboard and screen reader users. Currently it only responds to `enter`.

<img width="702" alt="Screen Shot 2021-06-30 at 2 26 57 PM" src="https://user-images.githubusercontent.com/14154792/124012516-3c5a7f00-d9af-11eb-9ec3-ad061f0b0200.png">
[Keyboard accessibility guidance from WebAIM](https://webaim.org/techniques/keyboard/)

**Comment by srsuddath** [comment]
Interesting. I had never heard the spacebar input requirement before. I can investigate and see about adding support for the spacebar press as well

let me clarify the intended functionality:
IF the user has the search button focused, then pressing the space button should initiate a search using the query populated in the input field

**Comment by joshkimux** [comment]
Clarification is correct 👍 
Space key is native to activating the HTML `button` and thus is expected behavior.

**Comment by srsuddath** [comment]
This ticket has been closed as part of the Search UI PR
